src/org/opensolaris/opengrok/web/api/controller/StatsController.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+ */
+package org.opensolaris.opengrok.web.api.controller;
+
+import org.json.simple.parser.ParseException;
+import org.opensolaris.opengrok.configuration.RuntimeEnvironment;
+import org.opensolaris.opengrok.web.Statistics;
+import org.opensolaris.opengrok.web.Util;
+
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+
+@Path("/stats")
+public class StatsController {
+
+    private final RuntimeEnvironment env = RuntimeEnvironment.getInstance();
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public String get() {
+        return Util.statisticToJson(env.getStatistics()).toJSONString();
+    }
+
+    @DELETE
+    public void clean() {
+        env.setStatistics(new Statistics());
```

Comment:
https://stackoverflow.com/questions/25970523/restful-what-should-a-delete-response-body-contain

Reply:
yes, but droping existing feature

Reply:
In this case I think it is redundant to always return `{}`. And as you mentioned, it should be as clean as possible; therefore, 2 requests one for delete and second for get would be more clear. Otherwise it does 2 things whereas it should do one.